### PR TITLE
enable AWS Backup

### PR DESCRIPTION
**Pull request by michaelwittig**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by michaelwittig** [comment]
test

**Comment by michaelwittig** [comment]
wait for long term test to succeed

**Review by andreaswittig — APPROVED** [review]

File: module.yml [hunk 1]
```diff
@@ -62,11 +72,63 @@ Resources:
       Size: !Ref Size
       # TODO support SnapshotId
       VolumeType: !If [HasIops, io1, gp2]
+  BackupVault: # cannot be deleted with data
+    Type: 'AWS::Backup::BackupVault'
+    Properties:
+      BackupVaultName: !Ref 'AWS::StackName'
+      Notifications: !If [HasAlertingModule, {BackupVaultEvents: [BACKUP_JOB_STARTED, BACKUP_JOB_COMPLETED, RESTORE_JOB_STARTED, RESTORE_JOB_COMPLETED, RECOVERY_POINT_MODIFIED], SNSTopicArn: {'Fn::ImportValue': !Sub '${AlertingModule}-Arn'}}, !Ref 'AWS::NoValue']
```
Comment (andreaswittig) [inline module.yml:81]: Do we really want to send notifications for every successful backup?

**Review by andreaswittig — APPROVED** [review]

**Comment by michaelwittig** [comment]
test

**Comment by michaelwittig** [comment]
test

**Pull request merged**